index.html

```diff
@@ -64,8 +64,11 @@
       <br>
       <a href="#" class="[ button ][ ffab before remove fa-arrow-left ]">Transition Remove</a>
       <a href="#" class="[ button ][ ffab remove fa-arrow-right ]">Transition Remove</a>
+      <hr>
+      <strong>Hover/Active State</strong>
       <br>
-      <a href="#" class="[ button ][ ffab static fa-arrow-right ]">No Transition (Static)</a>
+      <a href="#" class="[ button ][ ffab fa-arrow-right ]">With Hover/Active</a>
+      <a href="#" class="[ button ][ ffab fa-arrow-right no-hover-active ]">Without Hover/Active</a>
```

Comment:
That's what the code is already doing. If you set `$ffab-has-hover-active-state` to `false` this feature is turned off. The `no-hover-active` is a way to disable the feature on an individual component.

I haven't thought about adding the ability to add the feature on an individual component. Do you think that would be valuable functionality?